src/main/resources/interface-specification.yml

```diff
@@ -10923,6 +11002,33 @@ components:
       required:
         - origin
         - value
+    Tenants:
+      title: Tenants
+      type: object
+      properties:
+        results:
+          type: array
+          items:
+            $ref: '#/components/schemas/TenantEntry'
+        pagination:
+          $ref: '#/components/schemas/Pagination'
+      required:
+        - results
+        - pagination    
+    TenantEntry:
```

Comment:
May I suggest to rename this to `CompactTenant`?
`CompactTenant` is already used, but it's identical to `CompactOrganization`, so I'm suggesting these changes:

- current `CompactTenant` usage move to `CompactOrganization`
- rename `TenantEntry` to `CompactTenant`